agent/api/task/task.go

```diff
@@ -149,6 +149,8 @@ const (
 	// sysctlValueOff specifies the value to use to turn off a sysctl setting.
 	sysctlValueOff = "0"
 
+	// serviceConnectAttachmentType specifies attachment type for service connect
+	serviceConnectAttachmentType            = "ServiceConnect"
```

Comment:
Similar to the comment above. Lets create a new class - `TaskAttachment` and move the constants there.